Bukkit/src/main/resources/plugin.yml

```diff
@@ -131,7 +131,8 @@ permissions:
       plots.toggle.chat: true
       plots.set.biome: true
       plots.set.home: true
-      plots.set.alias: true
```

Comment:
Is there any way to keep this backwards compatible? I think it would be better to avoid changing (~removing~) the permission. Thoughts?

Reply:
@Sauilitired
what would be better for you addind the permisson again and use the 
plots.set.alias as obsolete permission and plots.set.alias as new permisson.
**or** 
leave the original permisson plots.set.alias and remove the new one plots.alias.set
Note: should we change the plots.alias.remove to plots.remove.alias or should we leave it as it was

Reply:
@Sauilitired
what would be better for you addind the permisson again and use the
plots.set.alias as obsolete permission and plots.set.alias as new permisson.
or
leave the original permisson plots.set.alias and remove the new one plots.alias.set
Note: should we change the plots.alias.remove to plots.remove.alias or should we leave it as it was